src/Containers/SearchContainer.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import SearchBar from "../Components/SearchBar";
+import axios from 'axios';
+
+class SearchContainer extends React.Component {
+    constructor(props){
+        super(props);
+        this.state= {
+            wordDetails:{}
+        }
+    }
+
+    getOneWordData = async (wordToSearch) => {
+        const result =  await axios.get(`https://wordsapiv1.p.rapidapi.com/words/${wordToSearch}`, {
+            headers: {
+                'x-rapidapi-host': 'wordsapiv1.p.rapidapi.com',
+                'x-rapidapi-key': '649dc644ddmshf772a2206d780e7p136ff6jsnd4074b64ab92'
```

Comment:
use these strings from a constants file

Reply:
also try passing the key as command line arg.
Remember, this key is now out in public because you have pushed it to github.

Reply:
What happens if API fails